cmd/otelcol/config/collector/agent_config.yaml

```diff
@@ -123,7 +123,11 @@ processors:
       #- action: insert
         #value: staging/production/...
         #key: deployment.environment
-
```

Comment:
nit: let's keep the space before the `exporters` section

Reply:
Changes committed

Reply:
@hughesjj Yes, only needed on the OOTB OTel dashboards